sql/_27_banana_qa/issue_5765_timezone_support/_08_current_date_function/_02_functions/cases/_02_addmonths.sql

```diff
@@ -22,86 +23,86 @@ set timezone 'America/Chicago';
 set @x1=(select adddate(current_date,interval 1 month));
 set timezone 'America/Los_Angeles';
 set @x2=(select add_months(current_date,1));
-select if(month(@x1)-month(@x2)=0,'ok','nok');
+select if(month(@x1)-month(@x2) in (0,1,-11), 'ok', 'nok');
 drop variable @x1,@x2;
 
 set timezone 'America/Chicago';
 set @x1=(select adddate(current_datetime,interval 1 month));
 set timezone 'America/Los_Angeles';
 set @x2=(select add_months(current_datetime,1));
-select if(month(@x1)-month(@x2)=0,'ok','nok');
+select if(month(@x1)-month(@x2) in (0,1,-11), 'ok', 'nok');
 drop variable @x1,@x2;
 
 set timezone 'America/Chicago';
 set @x1=(select adddate(curdate(),interval 1 month));
 set timezone 'America/Los_Angeles';
 set @x2=(select add_months(curdate(),1));
-select if(month(@x1)-month(@x2)=0,'ok','nok');
+select if(month(@x1)-month(@x2) in (0,1,-11), 'ok', 'nok');
 drop variable @x1,@x2;
 
 set timezone 'America/Chicago';
 set @x1=(select adddate(current_date(),interval 1 month));
 set timezone 'America/Los_Angeles';
 set @x2=(select add_months(current_date(),1));
-select if(month(@x1)-month(@x2)=0,'ok','nok');
+select if(month(@x1)-month(@x2) in (0,1,-11), 'ok', 'nok');
 drop variable @x1,@x2;
 
 et timezone 'America/Chicago';
```

Comment:
Please fix a typo together:
as is: 
et timezone 'America/Chicago';

to be:
set timezone 'America/Chicago';

Thank you.